airbyte-cdk/python/airbyte_cdk/sources/utils/schema_helpers.py

```diff
@@ -142,7 +142,8 @@ def check_config_against_spec_or_exit(config: Mapping[str, Any], spec: Connector
     try:
         validate(instance=config, schema=spec_schema)
     except ValidationError as validation_error:
-        raise Exception("Config validation error: " + validation_error.message)
```

Comment:
The reason why Exception here (instead of sys.exit) is that we have acceptance test that check if any traceback present on stderr in case of non-zero returncode. You should keep raising Exception just make sure the would be no sensitive information in output. e.g.
```python
 raise Exception("Config validation error: " + validation_error.message) from None
```